### ISSUE TITLE
`foreach': no implicit conversion of nil into String (TypeError

### ISSUE THREAD

**Issue by mictadlo** [wurmlab/flo#18]
Hi,
I ran in the following problem:
```
/apps/flo/gff_remove_feats.rb annotation_v2.gff3 > annotation_v2_cleaned.gff3 
apps/flo/gff_remove_feats.rb:15:in `foreach': no implicit conversion of nil into String (TypeError)
	from /apps/flo/gff_remove_feats.rb:15:in `<main>'
```

Did I miss anything?

Thank you in advance.

Best wishes,

Michal

**Comment by yeban** [comment]
You missed the first argument - the feature type to remove.
```
gff_remove_feats.rb gene input.gff > output.gff
```

**Comment by yeban** [comment]
Please don't close the issue. I will take this as a reminder to add a check to the script.

**Comment by mictadlo** [comment]
Thank you it worked

**Comment by yeban** [comment]
Closed in 3f3ce61673a898405ce138a141cebb56f370c64e
